Blueprints/BlueprintDefinitions/vs2022/AnnotationsServerless-Image/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -0,0 +1,154 @@
+# .NET Lambda Annotations Framework (Preview)
+
+The Lambda Annotations framework is currently in **preview**. Development for this framework is being done on the [aws/aws-lambda-dotnet](https://github.com/aws/aws-lambda-dotnet) repository. We would appreciate any feedback or feature suggestions on this library. 
+
+Lambda Annotations is a programming model for writing .NET Lambda function. At a high level the programming model allows
+idiomatic .NET coding patterns and uses [C# Source Generators](https://docs.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/source-generators-overview) to bridge the gap between the Lambda programming model to the more .NET idiomatic programming model.
+
+For example here is a simplistic example of a .NET Lambda function that acts like a calculator plus method using the normal 
+Lambda programming model. It respondes to an API Gateway REST API, pulls the operands from the resource paths, does the 
```

Comment:
```suggestion
Lambda programming model. It responds to an API Gateway REST API, pulls the operands from the resource paths, does the 
```

Reply:
Fixed also in the README.md for the NuGet package